.github/workflows/rust.yml

```diff
@@ -49,10 +49,10 @@ jobs:
       - uses: Swatinem/rust-cache@v1
 
       - name: Rustfmt
-        run: cargo fmt -- --check
+        run: cargo fmt --all -- --check
```

Comment:
Odd that --workspace doesn't work for fmt huh

Reply:
Yeah, strange they didn't deprecated it as well